src/Cvl/LDAPWrapper/LDAPWrapper.php

```diff
@@ -251,96 +253,199 @@ public function deleteDN($dn) {
 	}
 
 	/**
+	 * @deprecated Use createPerson instead
 	 * Creates a user in the $newUserDir and adds it to the required groups
+	 *
 	 * @param string $firstName
 	 * @param string $lastName
 	 * @param string $username
 	 * @param string $email
 	 * @param string $office
 	 * @param string $password
 	 * @param boolean $active
-	 * @return string The distinguished name of the created user, or null if the operation fails
+	 * @throws Exception
 	 */
 	public function createUser($firstName, $lastName, $username, $email, $office, $password, $active) {
-		$fullName = $firstName . ' ' . $lastName;
-		$info = array(
+		throw new Exception('createUser is deprecated, use createPerson instead');
+	}
+
+	/**
+	 * @deprecated Use editPerson instead
+	 * Edits the user with the given $currentUsername
+	 *
+	 * @param string $currentUsername
+	 * @param string $firstName
+	 * @param string $lastName
+	 * @param string $username
+	 * @param string $email
+	 * @param string $office
+	 * @param boolean $active
+	 * @throws Exception
+	 */
+	public function editUser($currentUsername, $firstName, $lastName, $username, $email, $office, $active) {
+		throw new Exception('editUser is deprecated, use editPerson instead');
+	}
+
+	/**
+	 * If cn attribute is present in $attrs remove it from the array and return it, otherwise return $defaultCN.
+	 *
+	 * @param array $attrs Passed by reference and may be modified
+	 * @param string $defaultCN
+	 * @return string
+	 */
+	protected function extractCN(&$attrs, $defaultCN) {
+		$commonName = $defaultCN;
+		if (isset($attrs['cn'])) {
+			$commonName = $attrs['cn'];
+			unset($attrs['cn']);
+		}
+		return $commonName;
+	}
+
+	/**
+	 * Get the user account control bits based on the $active flag.
+	 *
+	 * @param boolean $active
+	 * @return integer
+	 */
+	protected function getUserAccountControl($active) {
```

Comment:
Since this method is not extracting/getting data from any LDAP object, IMHO `calculateUserAccountCotrolBits` is less confusing name for this method...